audit/audit.cc

```diff
@@ -38,21 +39,44 @@ sstring audit_info::category_string() const {
     return "";
 }
 
-audit::audit(locator::shared_token_metadata& token_metadata,
+sstring audit_info::category_string() const {
+    return category_to_string(_category);
+}
+
+sstring tables_to_string(const std::map<sstring, std::set<sstring>>& tables)
```

Comment:
if it's only for logging audit_updatable_tables should be compatible with fmt library instead, same applies to statement_category